hack/lib/golang.sh

```diff
@@ -433,6 +433,50 @@ kube::golang::get_physmem() {
   echo 1
 }
 
+kube::golang::unhide_godeps() {
+  echo "Restoring Godeps to their rightful location"
+  mv "${KUBE_ROOT}/.Godeps" "${KUBE_ROOT}/Godeps/"
+  rm "${KUBE_ROOT}/_output/local/go/vendor"
+}
+
+kube::golang::hide_godeps() {
+  echo "Hiding godeps from GB"
+  mv "${KUBE_ROOT}/Godeps" "${KUBE_ROOT}/.Godeps"
+  ln -s "${KUBE_ROOT}/.Godeps/_workspace" -T "${KUBE_ROOT}/_output/local/go/vendor" >/dev/null 2>&1 || true
```

Comment:
So I don't think I actually need to hide Godeps, still on my to-test place. Also gb has some ummm, quirks, when it comes to symlinks. Work on filing some bugs right now.